data/anaconda.conf

```diff
@@ -46,6 +46,10 @@ physical_root = /mnt/sysimage
 # A path to the system root of the target.
 system_root = /mnt/sysroot
 
+# Should we install the network configuration?
+can_configure_network = True
```

Comment:
Not sure if the name of the value shouldn't be more specific. I mean `can_configure_network` is by name applicable even to installation environment not just to resulting system.

Maybe something like `can_install_network`?